squiggy/models/course.py

```diff
@@ -140,6 +140,7 @@ def get_advanced_asset_search_options(cls, course_id):
             SELECT id, title
             FROM categories
             WHERE course_id = :course_id
+            AND visible = True
```

Comment:
@mlincoln - Fyi, I typically write the above as `AND visible IS TRUE` but I believe the it'll work as is.

Reply:
Amended